rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,184 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.2
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka, bukka@php.net
+  * Status: Draft
+
+===== Introduction =====
+
+PHP uses its voting system to decide about changes to PHP. This was created mainly for user facing changes. It works
+well for that purpose but it is not that effective for deciding about purely technical changes that impact PHP
+internals and extension API's. It is not clear how it should resolve conflicts on the technical basis between
+contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes an introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src. Its primary but not only role is to resolve disagreements between contributors.
+
+==== Definitions ====
+
+The text contains various terms that need further clarification to prevent future wording dispute. Here are their
+definitions:
+
+  * php-src - Source code in https://github.com/php/php-src/.
+  * contributor - Anyone who has already contributed to php-src.
+  * core developer - Anyone with write commit access to php-src.
+  * dev branch - php-src branch consisting of PHP- prefix and major and minor version number (e.g. PHP-8.2).
+  * maintainer - person responsible for extension or SAPI that is listed in the php-src EXTENSION file.
+  * issue - php-src GitHub issue, including pull requests.
+  * PR - Pull request
+  * process changes - Any changes impacting the way how PHP project is managed and its bylaws.
+  * pull request - php-src GitHub pull request or issue.
```

Comment:
I've never seen anybody use the term "pull request" for issues

Reply:
This was actually changed by @Crell which I assumed is for easier pointing in the text but I agree that pull request should be just pull request and we could just leave issue as both.